pynopegl/init_header.py

```diff
@@ -167,6 +167,7 @@ def _pythonize_backends(backends: Mapping[str, Any]) -> Mapping[Backend, Any]:
         for cap_str, cap_data in backend_data["caps"].items():
             cap_id = Cap[cap_str.upper()]
             new_caps[cap_id] = cap_data
+        backend_data["id"] = Backend(backend_data["id"])
```

Comment:
It is not different than `backend_id`. The whole chunk is factored out into `_pythonize_backend()` later on and used by `_pythonize_backends()` and `ctx.get_backend()`. While it could be considered redundant while dealing with the backends dictionarry, I think it makes the behavior of the API consistent, ie: you can copy a backend from the backend dictionary and use it the same way as the one returned by `ctx.get_backend()`